web/ts/watch.ts

```diff
@@ -16,8 +16,36 @@ export class Watch {
 
 let ws: WebSocket;
 let retryInt = 5000; //retry connecting to websocket after this timeout
+let orderByLikes = false; // sorting by likes or by time
+
 const pageloaded = new Date();
 
+export function likeMessage(id: number) {
+    ws.send(
+        JSON.stringify({
+            type: "like",
+            id: id,
+        }),
+    );
+}
+
+export function sortMessages(messages) {
```

Comment:
This function should have a return type, I think

Reply:
`Array.prototype.sort()` sorts in place. Should we add `void` to such functions?

Reply:
From what I've seen for example at matrix `void` is usually added here to be more explicit and safe, but it is not really needed.

Reply:
I added `void`, since we have a bunch of such methods, do you think it would make sense to apply this everywhere or enforce it with eslint? @alexanderstephan

Reply:
Yes, that would be nice, see https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/explicit-function-return-type.md e,g.
But we could add that as well in a follow up.